docs/markdown/foundations/content_standards/ui_elements.md

```diff
@@ -0,0 +1,194 @@
+---
+title: UI Elements
+description: Here are a few guidelines we recommend you follow when writing near UI elements to help make things clear
+fullwidth: true
+---
```

Comment:
Need to remove these throughout the page

Reply:
@ayeshakmaz  - do you mean no descriptions?

Reply:
The line breaks

Reply:
Ooh that's the frontmatter

Reply:
We need em